kfs-core/src/main/java/edu/arizona/kfs/tax/businessobject/TransmitterRecord.java

```diff
@@ -0,0 +1,513 @@
+		}
+
+		if (!RecordUtil.hasCharacter(transmitterName1, 40)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Transmitter Name 1", transmitterName1, "Transmitter Name 1 is a required field. [limit = 40]"));
+		}
+		if (!RecordUtil.hasCharacterEmpty(transmitterName2, 40)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Transmitter Name 2", transmitterName2, "Transmitter Name 2 is too long. [limit = 40]"));
+		}
+
+		if (contactPhoneNumber == null) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Contact Phone Number", contactPhoneNumber, "Contact phone number is a required field."));
+		}
+		if (!RecordUtil.hasCharacterEmpty(contactEmail, 50)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Contact Email", contactEmail, "Contact email address is too long. [limit = 40]"));
+		}
+
+		if (!RecordUtil.hasCharacter(companyName1, 40)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company Name 1", companyName1, "Company Name 1 is a required field. [limit = 40]"));
+		}
+		if (!RecordUtil.hasCharacterEmpty(companyName2, 40)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company Name 2", companyName2, "Company Name 2 is too long. [limit = 40]"));
+		}
+
+		if (!RecordUtil.hasCharacter(companyMailingAddress, 40)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company Mailing Address", companyMailingAddress, "Company mailing address is a required field. [limit = 40]"));
+		}
+		if (!RecordUtil.hasCharacter(companyCity, 40)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company City ", companyCity, "Company city is a required field. [limit = 40]"));
+		}
+		if (!RecordUtil.isValidStateCode(companyState)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company State", companyState, "Company state is not a valid code. [limit = 2]"));
+		}
+		if (!RecordUtil.isValidZipCode(companyZipCode)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company Zip Code", companyZipCode, "Company zip code is not a valid code. [limit = 9]"));
+		}
+		if ("1".equals(foreignEntityIndicator) && !RecordUtil.isValidCountryCode(companyCountryCode)) {
+			errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company Country Code", foreignEntityIndicator, "Company country code is not a valid code. [limit = 2]"));

```

Comment:
Should be: 
`errors.add(new ElectronicFileException(transmitterTin, companyName1, ElectronicFileException.TRANSMITTER, "Company Country Code", companyCountryCode, "Company country code is not a valid code. [limit = 2]"));`